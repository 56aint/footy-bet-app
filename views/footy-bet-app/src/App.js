import logo from './logo.svg';
import './App.css';

function App() {
  return (
    <div className="App">
      FootyBet
    </div>
  );
}

export default App;
