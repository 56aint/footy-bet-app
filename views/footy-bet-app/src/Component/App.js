import React from 'react';
import '../Styles/App.css';
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
import MatchDay from './MatchDay';
import GetEvent from '../api/apiCalls/Components/EventCall';

function App() {
  return (
    <div className="App">
      <Router>
        <Switch>
          <Route exact path="/" component={MatchDay} />
          <Route exact path="/my-favourite-team" component={GetEvent} />
        </Switch>
      </Router>
    </div>
  );
}

export default App;
