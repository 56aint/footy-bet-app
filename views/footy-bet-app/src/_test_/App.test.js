import React from 'react';
import { render, screen } from '@testing-library/react';
import { MemoryRouter } from "react-router-dom";
import App from '../Component/App';


describe("App", () => {
  test('renders correctly', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <App />
      </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
  
  test("renders correctly", () => {
    const { getByText, getByTestId } = render(
      <MemoryRouter>
        <App />
      </MemoryRouter>
    );
    expect(getByText("Event Date")).toBeInTheDocument();
    expect(getByText(/Home/)).toBeInTheDocument();
    expect(getByText(/Away/)).toBeInTheDocument();
    expect(getByTestId("footy-event-id")).toBeInTheDocument();
    expect(getByTestId("footy-event-id")).toBeInTheDocument();
  });
});
