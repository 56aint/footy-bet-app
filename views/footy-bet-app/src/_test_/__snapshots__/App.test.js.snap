// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="footyevent"
    >
      <div
        class="container"
      >
        <h1>
          Football
        </h1>
        <div
          class="title-box"
        >
          <p
            class="startTime"
          >
            Event Date
          </p>
        </div>
        <div
          class="title-box"
        >
          <p
            class="teamsName"
          >
            Home-Team VS Away-Team
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
